src/components/GlobalSearch.jsx

```diff
@@ -10,40 +11,62 @@ function GlobalSearch({
   onClick,
   searchValue,
   handleClear,
-  img
+  img,
+  searchCount,
+  setSearchCount
 }) {
+  const handleErrorClose = () => {
+    setSearchCount(false);
+  };
   return (
-    <div className="flex relative">
-      <Textarea
-        val={searchValue}
-        className={inputClasses}
-        rows={4}
-        cols={0}
-        placeholder="Search by Style or Title or Brand"
-        onChange={onChange}
-      />
-      {searchValue && (
+    <>
+      <div className="flex relative">
+        <Textarea
+          val={searchValue}
+          className={inputClasses}
+          rows={4}
+          cols={0}
+          placeholder="Search by Style or Title or Brand"
+          onChange={onChange}
+        />
+        {searchValue && (
+          <Button
+            onClick={handleClear}
+            className="absolute top-[36%] right-[14%]"
+          >
+            <img
+              className="h-[12px] w-[12px]"
+              src={img}
+              alt="NavBarSwitchingIcon SVG"
+              data-testid="test-img"
+            />
+          </Button>
+        )}
         <Button
-          onClick={handleClear}
-          className="absolute top-[36%] right-[14%]"
+          className={buttonClasses}
+          onClick={onClick}
+          disabled={!searchValue}
         >
-          <img
-            className="h-[12px] w-[12px]"
-            src={img}
-            alt="NavBarSwitchingIcon SVG"
-            data-testid="test-img"
-          />
+          {" "}
+          {searchString}{" "}
         </Button>
+      </div>
+      {searchCount && (
+        <div className="flex items-center w-full p-4 bg-[#FC555619] text-grey-20 rounded shadow mt-2 border border-[#F95455] h-[75px]">
+          <div className="mr-2">
+            <img className="h-[21px] w-[23px]" src={Warning} alt="close SVG" />
+          </div>
+          <div className="text-sm font-[14px] text-[#4D4D4D]">
+            You have entered more than 500 styles. Please remove 501 styles
```

Comment:
Who gave this text ?

Reply:
Arun has given

Reply:
Text is not correct. Call him and get it corrected.